### ISSUE TITLE
When calling Universal Cookie.remove the default set options do not work

### ISSUE THREAD

**Issue by derekbassett** [bendotcodes/cookies#396]
If I configure a universal `Cookie` to map to a specific domain or path in the constructor. I would expect that I can remove that cookie without passing in additional information other than the name.
I enabled with react-cookie given remove/set a very similar I would expect the default values I pass into the constructor would also be included in the remove call.

```
cookies = new Cookies(null, {path: `/meow`});
cookies.set('testMeOut','value');
cookies.remove('testMeOut'); // I would expect the cookie set above would be removed.
```

**Comment by eXon** [comment]
Will be fixed in next release

**Comment by eXon** [comment]
Released in v6.1.2
